views/monitor.hbs

```diff
@@ -1,35 +1,62 @@
 {{!< default }}
-
-<main class="container monitor-homepage clear-header" data-page-label={{> analytics/landing_page_label }}>
-  <section class="row jst-cntr">
+<main class="container monitor-homepage clear-header {{#if featuredBreach}} featured-breach {{/if}} {{#if featuredBreach.IsSensitive}} sensitive-landing {{/if}}" data-page-label={{> analytics/landing_page_label }}>
+  <section class="row jst-cntr monitor-landing">
     <div class="col-12 flx-cntr landing-content">
-      {{#if featuredBreach}}
-        <h2 class="headline landing-headline">{{{ fluentNestedBold "was-your-info-exposed" breachName=featuredBreach.Title }}}</h2>
-      {{else}}
-        <h2 class="headline landing-headline">{{ getString "see-if-breached" }}</h2>
-      {{/if}}
-      {{#if featuredBreach.IsSensitive }}
-        <h3 class="subhead landing-subhead">{{ getString "find-out-if" }}</h3>
-        {{> feature-list variableClass="landing-feat-list" }}
-        {{> sign-up-button buttonValue="sign-up-for-alerts" variableClasses="landing-sign-up-btn"}}
-      {{else}}
-        <h3 class="subhead landing-subhead">{{ getString "find-out-what-hackers-know" }}</h3>
-        {{> forms/scan-email-form }}
-      {{/if}}
-    </div>
-  </section>
-</main>
-
-
-<div class="landing-bottom-bar">
-  <section class="article-list-container container">
-    <section class="row">
-      <div class="col-4">
-        {{> latest-breach }}
+      {{#makeLanding}}
+          <h2 class="headline landing-headline">{{{ headline }}}</h2>
+          <h3 class="subhead landing-subhead">{{ subhead }}</h3>
+      {{/makeLanding}}
+          {{#unless featuredBreach.IsSensitive}}
+            {{> forms/scan-email-form }}
+          {{/unless}}
+      {{#makeLanding}}
+          {{#if info}}
+            <section class="landing-sensitive-info">
+              {{#each info}}
+                <h3 class="sb-callout">{{ subhead }}</h3>
+                <p class="sb-callout-body">{{ body }}</p>
+              {{/each}}
+            </section>
+          {{/if}}
+      {{/makeLanding}}
       </div>
-      {{> take-back-control/article-list }}
     </section>
-  </section>
-</div>
 
-{{> sign-up-banners/monitor-features }}
+  {{#if featuredBreach}}
+    {{#if featuredBreach.IsSensitive}}
+    <div class="sb-top-container">
+      <div class="row no-vertical-padding sb-top-wrapper">
+        <div class="sb-top"><!--top of sign up call out--></div>
+      </div>
+    </div>
+     {{/if}}
+  {{else}}
+    <div class="latest-breach-headline-wrapper">
+      <h3 class="latest-breach-headline">{{{ getString "latest-breach" brandName="<span class='fxm'>Firefox Monitor</span>" }}}</h3>
```

Comment:
Not sure how localizing the `brandName` would work here.
I tried doing a <kbd>git grep "\-brand-name = "</kbd>, but it looks like `-brand-name` is just "Firefox" in most locales:

```sh
locales/en/app.ftl:-brand-name = Firefox
```

vs `-product-name` which is usually "Firefox Monitor":

https://github.com/mozilla/blurts-server/blob/db442bf6b24582517412f416fa5a1d54eca1e80f/locales/en/app.ftl#L6

... with a couple exceptions:

```sh
git grep "\-product-name = " | grep -v " = Firefox Monitor"

locales/km/app.ftl:-product-name = កម្មវិធី​ត្រួតពិនិត្យ Firefox
locales/km/email-strings.ftl:-product-name = កម្មវិធី​ត្រួតពិនិត្យ Firefox
locales/ml/app.ftl:-product-name = ഫയർഫോക്സ് മോണിറ്റർ
locales/ml/email-strings.ftl:-product-name = ഫയർഫോക്സ് മോണിറ്റർ
```

Not sure if it's worth renaming the arg to `productName` for consistency, but that's super minor nit.